Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "tantivy-tokenizer-tiny-segmenter"
-version = "0.1.0"
+version = "0.1.1"
```

Comment:
Can you remove the version bump of tantivy-tokenizer-tiny-segmenter here? I just prefer to keep these independent of PRs